code/controllers/subsystem/persistance.dm

```diff
@@ -21,6 +21,9 @@ SUBSYSTEM_DEF(persistence)
 	//var/list/obj/item/storage/photo_album/photo_albums
 	var/list/obj/structure/sign/painting/painting_frames = list()
 	var/list/paintings = list()
+	var/list/obj/structure/statuebase/pedestals = list()
+	var/list/statues = list()
+
```

Comment:
Probably need to rename this file `persistence.dm` and change it in `paradise.dme` as well.

Reply:
Blame tg not knowing how to english, and me not noticing